chainspecs/mainnetSpec.json

```diff
@@ -11,6 +11,7 @@
     "tokenSymbol": "CTC"
   },
   "consensusEngine": null,
+  "grandpaInitialAuthorities": [],
```

Comment:
We should discuss a plan for how we want to handle this during the release process. I assume we'll need to update nodes, stake them, update the chainspec, build a new docker image, and then publicly release that updated image. Is that right? Any way to make the process simpler?

Reply:
The idea is that the first set of validators would be predetermined. It's the only way to really make it workable IMO. So the idea is you would:
- Generate some accounts for our initial authorities (/ collect them from third-party validators that we trust) - this can be done pre upgrade
- Update chainspec to include the chosen authorities
- Publish new docker image (the first with the PoW/PoS switch support)
- Update the nodes
- Do the PoS switch when we're ready (and in the `switchToPos` extrinsic call we would specify the initial authorities)

Reply:
At what point would the generated accounts become associated with the PoW nodes?